src/file-1.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:36781/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:36781/project/issues?id=sample&pullRequest=5413&issues=AWcRSB0mETmP-yeb5ScS&open=AWcRSB0mETmP-yeb5ScS)

[ISSUE_KEY]: AWcRSB0mETmP-yeb5ScS